packages/storybook/src/stories/grid-rows.stories.tsx

```diff
@@ -919,6 +919,19 @@ export function SwitchVirtualization() {
   );
 }
 
+export function DeferRenderingSnap() {
```

Comment:
I'm not sure if we need a story for this but I guess it doesn't hurt.

Reply:
I have the test failing on safari browserstack but working locally, so I wonder if a snap is enough 🤔

Reply:
+1 for removing the snap

Reply:
Does it hurt? Why do we have snap if we can't use them